content/intro-to-storybook/react-native/en/simple-component.md

```diff
@@ -121,7 +121,7 @@ Now that we've set up the basics lets re-run `yarn storybook` and see our change
 You should see a UI that looks like this:
 ![a gif showing the task component in storybook](/intro-to-storybook/react-native-task-component.gif)
 
-You can use the Sidebar tab to swap between stories, the Canvas tab to see the current story and the addons tab to interact with args and actions.
+You can use the menu at the bottom of your screen to open the navigation menu and tap to swap between stories, then you can tap away or drag down to dismiss the menu. You'll find addons by pressing the icon on the far right of the bottom bar.
```

Comment:
@dannyhw small item here if you don't mind.
```suggestion
You can use the menu at the bottom of your screen to open the navigation menu and tap to swap between stories. Then, you can tap away or drag down to dismiss the menu. You'll find addons by pressing the icon on the far right of the bottom bar.
```

Reply:
adjusted it, thanks!